index.js

```diff
@@ -22,8 +25,27 @@ app.use(cors());
 app.use(morgan('combined'));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
-
+app.use(session({
```

Comment:
@Dubby20 why are we using sessions again?

Reply:
@9jaswag we still have to use a session to be able to implement twitter login

Reply:
@Dubby20 if we have ti, lets skip Twitter login

Reply:
@Dubby20 ??

Reply:
Hmmm. Okay but I have to let the PO know